source/user/installation-guide/troubleshooting.md

```diff
@@ -3,20 +3,37 @@ Troubleshooting
 
 ## The live system does not boot.
 
+### If Secure Boot is enabled, in BIOS settings for your computer, disable it.
+
+Support for Secure Boot is [not yet integral to FreeBSD](https://wiki.freebsd.org/SecureBoot).
+
 ### Your hardware may not be supported.
 
-GhostBSD is based on the FreeBSD Stable branch and hardware support can change often. Hardware support is generally good, however, some more recent hardware may have issues. You can try to boot the live system on another computer to determine if the issue is related to your hardware or with the bootable USB itself.
+GhostBSD is based on [the STABLE branch](https://www.freebsd.org/where/#helptest) of FreeBSD.
+Hardware support is generally good, however, there may be issues with some modern hardware.
+
+To determine whether the issue relates to hardware, or to the USB drive:
+
+- try to boot from the same drive on a different computer.
 
-Please refer to [FreeBSD Hardware Compatibility](https://docs.freebsd.org/en/books/faq/#hardware) for more information on indiviual components.
+For information about hardware components, please see FreeBSD information for the _RELEASE_ that corresponds to the version of _STABLE_ used by GhostBSD.
+
+You can run `ghostbsd-version -kv` to show the version number. As an example, _**14**0**1**502_ was from the _14.1-STABLE_ branch, and **hardware information** is amongst the pages at <https://www.freebsd.org/releases/14.1R/>.
 
 ### The installation media may be corrupted.
 
-Ocassionally, ISO files may become corrupted during download or during the creation of a bootable USB flash drive. Please refer to the [Getting started](getting-started.md) guide to download a new ISO file and create a new bootable USB flash drive.
+Corruption may occur during download of an installer image file, or during creation of a bootable USB flash drive.
+Please refer to the [Getting started](getting-started.md) guide to download a new image and create a new bootable USB flash drive.
 
 ## The live system does not reach a graphical desktop environment.
 
-### The installation ISO may be outdated or require general bug fixes.
+…
+
+### The installation ISO may be outdated.
+
+Try creating a bootable USB using the [latest image](https://www.ghostbsd.org/download).
+GhostBSD is a rolling release and ocassionally updates and fixes are pushed to the latest build before a new official image is created.
```

Comment:
```suggestion
```

You might remove this sentence.

As things have modernised: will there _ever_ be more than a single version at <https://www.ghostbsd.org/download>? 

At the time of writing: 

- Official and Community images are both `24.10.1`
- there is no separate mention of a _latest_ (with a superior version).

Reply:
This part is outdated. We don't build the latest ISO like before.